src/languages/pl.ts

```diff
@@ -5952,7 +5952,7 @@ const translations = {
         },
         statements: 'Oświadczenia',
         unapprovedCash: 'Niezatwierdzone środki pieniężne',
-        unapprovedCompanyCards: 'Niezatwierdzone karty firmowe',
+        unapprovedCards: 'Niezaakceptowane karty',
```

Comment:
NAB: The Polish translation changed from "Niezatwierdzone karty firmowe" (Unapproved company cards) to "Niezaakceptowane karty" (Unaccepted cards). This introduces two issues:

1. Semantic Inconsistency: "Niezaakceptowane" (unaccepted) has a different meaning than "niezatwierdzone" (unapproved)
2. Cross-Language Inconsistency: Other languages maintained the "unapproved" concept:
•  EN: "Unapproved cards" 
•  DE: "Nicht genehmigte Karten" (Unapproved cards)
•  FR: "Cartes non approuvées" (Unapproved cards)
•  IT: "Carte non approvate" (Unapproved cards)

```suggestion
        unapprovedCards: 'Niezatwierdzone karty',
```

cc @luacmartins

Reply:
Updated!